### ISSUE TITLE
Import Task Submitted - typo and maybe rewording needed; no notification after import task completes

### ISSUE THREAD

**Issue by julienlim** [Tendrl/ui#550]
Typo
- "... submitted.  you will be ..." ==> "... submitted.  You will be ..."

Rewording (to remove the word "new")
"... the new cluster is ready ..." ==> "... the cluster is ready ..."

There seems to be no alert/notification generated when import cluster has completed (especially if it failed).

<img width="1280" alt="screen shot 2017-08-30 at 3 20 20 pm" src="https://user-images.githubusercontent.com/5589919/29941133-8060a67e-8e5f-11e7-9073-2072bc26a2f6.png">

@Tendrl/tendrl_frontend @gnehapk @Tendrl/tendrl-core @Tendrl/qe @mcarrano

**Comment by gnehapk** [comment]
We will care of typo. @julienlim Please add a RFE issue in backend component for generating notification when import cluster has completed.

**Comment by cybernth** [comment]
@julienlim, @gnehapk , support for notification on job status changes are added. It will not be specific to any job, but generic notifications with job-id.

@gnehapk , you can close this issue if the typo is fixed

**Comment by gnehapk** [comment]
@julienlim @nthomas-redhat typo is fixed.  Hence, closing this issue.
